openedx/core/djangoapps/credentials/api/v1/views.py

```diff
@@ -1,3 +1,55 @@
 """
 Credentials API views (v1).
 """
+from django.contrib.auth.decorators import login_required
+from django.contrib.auth.models import User
+from django.http import HttpResponse, HttpResponseForbidden, HttpResponseBadRequest
+import json
+
+
+@login_required()
+def generate_program_credential(request):
+    user = request.user
+    if not user.is_staff:
+        return HttpResponseForbidden()
+
+    program_id = request.GET.get('program_id')
+    if not program_id:
+        return HttpResponseBadRequest()
+
+    usernames = request.GET.get('users')
+    if not usernames:
+        return HttpResponseBadRequest()
+
+    usernames = usernames.split(',')
+    whitelist = request.GET.get('whitelist')
```

Comment:
`is_whitelist` would be slightly clearer.  I also think it would be helpful to establish the defaults here in these `.get` calls, so it is obvious to the reader what the values will be if the query parameters are omitted.